File: nivel-01/ex-03/juros_simples.py
#-*- coding:utf-8 -*- 

def calcula_juros(capital, taxa_mensal, meses):
	juros = capital * taxa_mensal * meses   
	total = capital + juros
	return total

assert 116 == calcula_juros(100, 0.04, 4)

File: nivel-01/ex-03/juros_simples.c
#include <stdio.h>
#include <assert.h>

//
// Calcula o juros simples
//
float juros_simples(float capital, float taxa_mensal, int meses) {
	int juros = capital * taxa_mensal * meses;
	return juros;
}

int main (){
    // Este é o nosso teste.
    // Sabemos que 16000 * 0.04 * 4 = 2560, logo
    // a variável `juros` deve ter o valor igual a 2560.
    assert(2560 == juros_simples(16000, 0.04, 4));
    return 0;
}